.gitattributes

```diff
@@ -1 +1,3 @@
 * text=auto eol=lf
+
+buf.yaml export-ignore
```

Comment:
Do we also want to exlclude package.json and package-lock.json as was reported on the [issue](https://github.com/open-telemetry/opentelemetry-proto/issues/604)? @marcalff you opened the issue, any thoughts?